Sources/NIOSSL/SSLErrors.swift

```diff
@@ -16,14 +16,33 @@
 
 /// Wraps a single error from BoringSSL.
 public struct BoringSSLInternalError: Equatable, CustomStringConvertible {
-    let errorCode: UInt32
-
-    var errorMessage: String? {
-        // TODO(cory): This should become non-optional in the future, as it always succeeds.
-        var scratchBuffer = [CChar](repeating: 0, count: 512)
-        return scratchBuffer.withUnsafeMutableBufferPointer { pointer in
-            CNIOBoringSSL_ERR_error_string_n(self.errorCode, pointer.baseAddress!, pointer.count)
-            return String(cString: pointer.baseAddress!)
+    private enum Backing: Hashable {
```

Comment:
Doesn't matter much but this is `Hashable` and `BoringSSLInternalError` is only `Equatable` -- do we want these to line up?

Reply:
For now I didn't think it was worth making the change to the public type.